shared/desktop/yarn-helper/index.js

```diff
@@ -26,11 +28,27 @@ const commands = {
   postinstall: {
     code: () => {
       // Nothing right now
+      // storybook uses react-docgen which really cr*ps itself with flow
+      // I couldn't find a good way to override this effectively (yarn resolutions didn't work) so we're just killing it with fire
+      makeShims()
     },
     help: '',
   },
 }
 
+function makeShims() {
+  const root = path.resolve(__dirname, '..', '..', 'node_modules', 'babel-plugin-react-docgen')
+
+  try {
+    fs.mkdirSync(root)
+  } catch (_) {}
+
+  try {
+    fs.writeFileSync(path.join(root, 'package.json'), `{"main": "index.js"}`)
+    fs.writeFileSync(path.join(root, 'index.js'), `module.exports = function(){return {};};`)
```

Comment:
Nothing expects this to exist?

Reply:
?

Reply:
I mean that you're effectively removing this file, right? so why doesn't that break anything else?

Reply:
only story book needs this so this plugin does nothign now. we don't use it. also it uses lib/index.js and not index.js